v3/wrap/client-config.ts

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
Random thought: what do you think about naming this file "polywrap.config.ts"?

Reply:
i feel like that name might be confusing with any toolchain manifest file :thinking: but I am not close to that tho